assets/questions/unknownAlphabet.js

```diff
@@ -0,0 +1,321 @@
+        "kmima",
+        "kmimd",
+        "kmiej",
+        "kmies",
+        "kmrbm",
+        "kmrbr",
+        "kmrus",
+        "kmrut"
+      ],
+      allowedOutputs: ['ljhmezirxygadoqckfsvntwbup']
+    }],
+    buggyOutputTests: [],
+    performanceTests: []
+  }, {
+    instructions: [
+      [
+        'Next, double-check your code to make sure it handles unexpected cases',
+        ', such as there\'s no solution to the test case or there\'re more ',
+        'than one answers.'
+      ].join(''),
+      [
+        'For the test case that has no solution, just return "". For the test ',
+        'case that has multiple answers, return the answer has the smallest ',
+        'lexicographic order in English alphabet.'
+      ].join(''),
+      'Example dictionary:',
+      '[a, b, a]',
+      'Alphabet is: ""',
+      '[cba]',
+      'Alphabet is: "abc"'
+    ],
+    prerequisiteSkills: ['Topological Sorting', 'String Manipulation'],
+    acquiredSkills: ['String Manipulation'],
+    inputFunctionName: null,
+    outputFunctionName: null,
+    mainFunctionName: 'findDictionary',
+    correctnessTests: [{
+      input: [
+        "a",
+        "b",
+        "a"
+      ],
+      allowedOutputs: ['']
+    }, {
+      input: [
+        "c",
+        "a",
+        "b",
+        "a"
+      ],
+      allowedOutputs: ['']
+    }, {
+      input: [
+        "cab",
+        "bc"
+      ],
+      allowedOutputs: ['acb']
+    }],
+    buggyOutputTests: [{
+      buggyFunctionName: 'AuxiliaryCode.wrongInNoSolution',
+      messages: [
+        [
+          'What if the test case is ["c", "a", "b", "a"]?',
+          'And what answer your code will generate for this test case?'
+        ].join(''),
+        [
+          'The answer to that case should be a empty string. But what your code',
+          ' have returned?'
+        ].join('')
+      ]
+    }],
+    performanceTests: []
+  }, {
+    instructions: [
+      [
+        'Finally, modify your code to handle strings with a mix of lowercase ',
+        'and uppercase letters. Assume that a lowercase and uppercase version ',
+        'of a letter is the same with regards to lexicographic ordering. Your ',
+        'answer, though, should be lowercase.'
+      ].join(''),
+      'Example dictionary:',
+      '[b, Ba]',
+      'Alphabet is: "ab"'
+    ],
+    prerequisiteSkills: ['Topological Sorting', 'String Manipulation'],
+    acquiredSkills: ['String Manipulation'],
+    inputFunctionName: null,
+    outputFunctionName: null,
+    mainFunctionName: 'findDictionary',
+    correctnessTests: [{
+      input: [
+        "ab",
+        "Aba",
+        "b"
+      ],
+      allowedOutputs: ['ab']
+    }],
+    buggyOutputTests: [{
+      buggyFunctionName: 'AuxiliaryCode.wrongInUppercase',
+      messages: [
+        [
+          'Think about this test case in your head: ["a", "Ab", "b"]'
+        ].join(''),
+        [
+          'The answer to the test case ["a", "Ab", "b"] should be "ab". Did ',
```

Comment:
I think this might be better served as two messages:

"How does your code handle uppercase letters?"

and then some message about not returning uppercase letters.